src/scss/index.scss

```diff
@@ -70,3 +70,11 @@ p {
 
     @apply rounded;
 }
+
+p > ul {
```

Comment:
can yuo also add the other styles we had with the ordered list? just in case we have an ordered list in the future we dont need to add extra style